sympy/printing/tests/test_ccode.py

```diff
@@ -119,6 +118,54 @@ def test_ccode_boolean():
     assert ccode((x & y) | z) == "z || x && y"
     assert ccode((x | y) & z) == "z && (x || y)"
 
+def test_ccode_Eq():
+    from sympy import Eq
+    from sympy import Le
+    x,y,z = symbols('x,y,z')
+    expr = Eq(x, Le(y,z))
+    assert ccode(expr) == "x == y <= z"
```

Comment:
no, let's not assert this.  I suspect its may not be correct.  Just `x == y` please.